packages/excalidraw/element/bounds.ts

```diff
@@ -75,23 +75,12 @@ export class ElementBounds {
     ) {
       return cachedBounds.bounds;
     }
-    const scene = Scene.getScene(element);
-    const bounds = ElementBounds.calculateBounds(
-      element,
-      scene?.getNonDeletedElementsMap() || new Map(),
-    );
+    const bounds = ElementBounds.calculateBounds(element, elementsMap);
 
-    // hack to ensure that downstream checks could retrieve element Scene
-    // so as to have correctly calculated bounds
-    // FIXME remove when we get rid of all the id:Scene / element:Scene mapping
-    const shouldCache = !!scene;
-
-    if (shouldCache) {
-      ElementBounds.boundsCache.set(element, {
-        version: element.version,
-        bounds,
-      });
-    }
+    ElementBounds.boundsCache.set(element, {
```

Comment:
do we need to set and retrieve from cache?

Reply:
of course, otherwise getting bounds would be a major bottleneck during hot paths.

Reply:
as for whether removing the `shouldCache` check won't break anything, I'm not sure. Can't reproduce the bug that this was fixing, but there may have been some edge case I don't recall.

Reply:
Yes my question was whether we need the check shouldCache anymore, as per the logic its not needed anymore